test/inductor/test_fp8.py

```diff
@@ -32,6 +33,17 @@
 FP16_MAX_POS: float = torch.finfo(torch.float16).max
 EPS: float = 1e-12
 
+# fp8 data types for inductor based fp8 tests. This can be different
+# than the one used in eager mode.
+f8_type_pair = (torch.float8_e4m3fn, torch.float8_e5m2)
+if torch.version.hip:
+    arch = torch.cuda.get_device_properties(0).gcnArchName
+    if "gfx94" in arch:
+        # for gfx942, use fnuz data type.
+        f8_type_pair = (torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)
+    elif "gfx120" in arch:
```

Comment:
```suggestion
    else:
```
gfx942 is the only gfx with fnuz support, so just use 'else' here.

Reply:
never mind, I see what you're doing now.